### ISSUE TITLE
Turtle test case turtle-eval-lists-04 fails

### ISSUE THREAD

**Issue by melgi** [melgi/cturtle#5]
The Turtle test case [turtle-eval-lists-04](http://w3c.github.io/rdf-tests/turtle/turtle-eval-lists-04.ttl) gives a different model than expected when parsed with `-f=nt`.

**Comment by melgi** [comment]
This is fixed, output:

```
_:bJU56M9QX7DSN6MR0-0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil> .
_:bJU56M9QX7DSN6MR0-0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil> .
_:bJU56M9QX7DSN6MR0-1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil> .
_:bJU56M9QX7DSN6MR0-1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil> .
_:bJU56M9QX7DSN6MR0-0 <http://www.w3.org/2013/TurtleTests/p> _:bJU56M9QX7DSN6MR0-1 .
```

**Comment by josd** [comment]
Bedankt Giovanni en alle testen zijn nu OK:
- running Turtle tests http://w3c.github.io/rdf-tests/turtle/ and
  passed 298 out of 298 tests
  http://josd.github.io/eyeearl/earl-report.ttl

Jos

On Fri, Apr 15, 2016 at 4:20 PM, Giovanni Mels notifications@github.com
wrote:

> Closed #5 https://github.com/melgi/cturtle/issues/5.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/melgi/cturtle/issues/5#event-628989779

## 

Jos De Roo, https://twitter.com/josderoo
